File: Beecrowd/Beginner/1101 - 1190/1189 - Left Area/C/1189_LeftArea.c
#include <stdio.h>

int main() {
    char operation;
    double matrix[12][12], sum = 0;
    scanf("%c", &operation);

    for (int i = 0; i < 12; i++) {
        for (int j = 0; j < 12; j ++) {
            scanf("%lf", &matrix[i][j]);

            if (i > j && i + j < 11) {
                sum += matrix[i][j];
            }
        }
    }

    if (operation == 'S') printf("%.1lf\n", sum);
    else printf("%.1lf\n", sum / 30);

    return 0;
}

File: Beecrowd/1 - Beginner/1101 - 1190/1189 - Left Area/Python/1189_LeftArea.py
sum = 0

operation = input()

for i in range(12):
    for j in range(12):
        number = float(input())
        if i > j and i + j < 11:
            sum += number
            
if operation == 'S':
    print(f"{sum:.1f}")
else:
    print(f"{sum / 30:.1f}")
